src/shared/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,21 @@
+import { validateToken } from "../auth/validateToken.js"
+
+const authMiddleware = (req, res, next) => {
+
+    const token = req.headers['authorization'];
```

Comment:
why not
```
    const { authorization } = req.headers;
```

Reply:
Vai dar ruim para requests que não tem token, faz isso nao